### DISCUSSION TITLE
3rd party exporters

### DISCUSSION THREAD

**Discussion by zfmq** [prometheus-operator/kube-prometheus#878] [Q&A]
Is there a tutorial on how to bundle more exporters? https://prometheus.io/docs/instrumenting/exporters/

Thanks!

**Comment by paulfantom** [comment]
Generally, exporters should be run close to the application. However, if you want to run it with kube-prometheus, you install it in similar way to how we handle blackbox_exporter installation.

**Comment by kehao95** [comment]
Hi @zfmq  you can check my repo here. https://github.com/kehao95/helm-prometheus-exporter
It's an helm template which helps you to install almost any exporters to work with prometheus-operator. I've been using it to deploy about 10 different exporters in my clusters. So it should be flexible enough.
